### ISSUE TITLE
Question: Would we be able to use modified VLM model code with the existing testing infrastructure?

### ISSUE THREAD

**Issue by bencondemi** [worldcuisines/worldcuisines#27]
Hello,

I am a student hoping to use this benchmark for a class's final project. If I were to modify an existing VLM, perhaps by adding fine-tuning for adversarial contexts and/or location prediction, would I be able to run the existing test-suite with that modified VLM code? If so, could you please lay out what that process would look like? If not, could you please spitball with me on alternative directions of things that I could do with this benchmark for my project? 

Thanks!

**Comment by gentaiscool** [comment]
Hi @bencondemi yes you can, please refer to this file https://github.com/worldcuisines/worldcuisines/blob/main/evaluation/src/llama.py you can modify the file and adjust accordingly to how you call your VLMs.

**Comment by bencondemi** [comment]
Great, thank you for your reply!

Can I also verify that your evaluation (WC-VQA) of the various models in your paper using Test Large (60k) was zero-shot testing in the sense that you did not expose the models to the Train (1M) set before running inference on the test set? Thanks again.

**Comment by gentaiscool** [comment]
Yes that is correct. The evaluation done in the paper is all zero shot. The training data is created to support future research.
